### DISCUSSION TITLE
记一次被老师体罚 | /posts/punisher/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#644] [Announcements]
https://yuanfan.rbind.io/posts/punisher/

**Comment by meeeeeeeeo** [comment]
哎，面对过去，面对未来

**Reply by earfanfan** [reply]
嘿，放心，其实我只隐约记得当时恐怖的氛围，具体感受已经忘得差不多了。好像是阿德勒的个体心理学有种说法，过去是由现在决定的。大意是说如何面对过去的创伤记忆是由现在决定的，是沉浸伤痛，还是走出来，决定权在于自己。

**Comment by cube5887** [comment]
看到您的文章，想起了我的初中班主任老师，不过是一个快30岁的女老师，她比较爱健身运动，打网球，平时惩罚学生就是打我们屁股或手心，如果是打屁股的话，在班里，需要手扶着黑板，腰塌下去，高撅起屁股，然后老师抡圆了胳膊抽下去，声音巨响，班里有的女生都要捂耳朵，要是打在穿校裙的女生上，校服裙子甚至会飞起来，直接走光，严重一点不管男生女生会让你把外面裤子脱掉打，冬天一定会这样，有时因为纪律或者成绩问题，甚至会被老师叫到办公室打你，有可能会让你脱下所有裤子把屁股露出来打，如果她用的是竹条一下就出血印子，会直接鼓起一道痕，两下打同一位置甚至会破皮，渗血，有时会用戒尺或者凳子面上拆下的木板条，打完屁股会红肿淤青，上厕所都是煎熬，有的时候被打的男女学生之间甚至会看到屁股，又很尴尬...

**Reply by earfanfan** [reply]
天啦，这个老师好可怕，我就小时候那次全班被打印象特别深刻，甚至就连写博客时都不愿意过多回忆细节，简直不敢想象你们那时候总是被打是撒样。不得不说，以前是老师承担了部分本该属于家庭的教导职责，导致那时候全社会都太纵容老师了。而且以前人们的思想观念又有“不打不成材”这种的，农村甚至还有“打是亲骂是爱”的说法，总之就是在人们表达能力普遍有限的年代里，这种打骂教育受到极大追捧，而且还在“孝道”观念的加持下，从小孩的角度看就是只要大人们有了家长或老师的身份，就有天经地义的权利可以打骂小孩，完全没有人追究这种暴力对待小孩的方式有时候仅仅只是因为大人无法管理好自己的情绪。

其实就算不是我本人被打，我觉得看着别的同学被打的那种氛围也很可怕。

**Reply by cube5887** [reply]
所以我一直难以释怀，有时就会想起这段经历，而且正如你所说，即使不是自己挨打，看到别人挨打一样会恐惧害怕，导致长期在一个压抑的环境下学习成长，那时候在农村，居然都认为老师的教育方式是没有问题的，如果有人明确不想让老师打，反而会不合群，甚至会被扣上不尊重老师的帽子，更甚至还会被家长打一顿

**Reply by earfanfan** [reply]
其实我觉得很少有人的思想能够跨越时代的，那个时候主流的社会思想是那样子的，不论家长、老师还是孩子都接受了那些规训，今天上班乘电梯的时候我顺嘴问两个同事小时候有没有被打过，一个描述小时候被打的惨状比如直接被拎板凳或竹条抽打，另一个却说这是“屁大点事”，我不知道他们现在是否还认同“打孩子的政治正确”，不过随着时代变化人们的需求也变了，现在打孩子的思想观念已经发生改变，可是孩子们却在一个更压抑的环境下学习成长了，还是有很多不同形式的暴力顺着一些名为天经地义的规训存在着。

我只记得小学时候被打，初中时候没被打过，不过看你描述的那些，初中的时候学生们的性别意识已经很清晰了还那样打，感觉留下的心理阴影会特别特别重。你提到的那位老师肯定不会为自己的所作所为有什么反省，可能还会为自己教出很多学生而得意，唉。讲真，其实我也不知道一直难以释怀的事应该如何释怀，容我请教一下小时候也被打过的一位朋友吧。

@yufree 三水，你好，我小时候也留下一些阴影在心灵深处，长大以后会逐渐淡忘，可是也会有些时机令我想起，我现在的处理方式就是和阴影保持距离，你比我们大几岁呢，好像也一向比我更看得开，可不可以向我们传授一些经验，或者给我们讲讲道理。

**Reply by yufree** [reply]
解决不了也不需要解决，有些事想起来就生气，越想越气，通常睡一觉或转移注意力干点其他事就忘了，但再想起来也是气，当然人类大脑也会倾向忘记不愉快并美化回忆。普通人就是有喜怒哀乐，会纠结过往，对抗人性的代价是很高的，但又不是所有人都要扫清所有心理阴影，那是修仙成圣，要渡劫上十字架的。通常我出现这种状态后会立马启动警报：你是不是工作量不饱和了？然后马上会有一堆事冒出来需要做，一直向前看，自然少回头。

**Reply by earfanfan** [reply]
@yufree 

实不相瞒，我最近冒出来一个想法，由于过去许多年会做噩梦被追杀，突然有一天我就想要不下回遇到干脆回头看看，反正我在梦里是不可能被杀死的。结果前两天又做噩梦，我根本还是连梦境和现实都分不清，第一反应就是好害怕、快躲快跑，根本就想不起要回头。看完三水你说的这些，虽然我还是很好奇梦里被追杀的时候回头看会看到什么，但是莫名其妙地感觉心底里多了一些温暖的释怀。谢谢三水，你真好。

**Reply by cube5887** [reply]
@earfanfan 那个老师据说后来还升职调到更好的学校了，因为她连续好几届教出来的成绩都很优秀，其实都是棍棒与家长守旧观念加持下的产物，用一时的成绩，遵守纪律，换来了我们一辈子的阴影，而且她当时在我们学校还是“教师劳模”，许多老师都会向她请教学习，如果说暴力是不合法的，但在部分家长与老师的观念里，由他们来实施就是合情合理的

**Comment by jax-169** [comment]
呜呜呜呜呜呜呜呜呜X﹏X，我就是一个刚毕业的小学生，我也被打过，晚上睡觉都梦到过班主任发试卷的时候拿戒尺杀人

**Reply by jax-169** [reply]
听别人说我上的那所初中的老师也会打人

**Reply by cube5887** [reply]
那你现在还好吗
